caluma/extensions/events/work_item.py

```diff
@@ -135,17 +136,25 @@ def send_new_work_item_mail(sender, work_item, user, **kwargs):
 
 
 @on(post_create_work_item, raise_exception=True)
+@on(post_reopen_case, raise_exception=True)
 @filter_events(
     lambda sender: sender
     in [
         "post_complete_work_item",
         "post_skip_work_item",
         "case_post_create",
         "post_redo_work_item",
+        "post_reopen_case",
     ]
 )
```

Comment:
Just for confirmation - this looks kinda redundant, as we're only triggering on `post_create_work_item` and `post_reopen_case` anyway.. Could this be removed?